sdk/identity/identity/src/credentials/workloadIdentityCredential.ts

```diff
@@ -87,11 +116,18 @@ export class WorkloadIdentityCredential implements TokenCredential {
     options?: GetTokenOptions,
   ): Promise<AccessToken | null> {
     if (!this.client) {
-      const errorMessage = `${credentialName}: is unavailable. tenantId, clientId, and federatedTokenFilePath are required parameters. 
-      In DefaultAzureCredential and ManagedIdentityCredential, these can be provided as environment variables - 
+      const errorMessage = `${credentialName}: is unavailable. tenantId, clientId, and either federatedTokenFilePath or serviceConnectionId are required parameters. 
```

Comment:
Same here, we should use `tokenFilePath` in the message if that's the name of the parameter

Reply:
oh let me check

Reply:
updated the errors with the correct field name